lxd/api_project.go

```diff
@@ -334,11 +333,9 @@ func projectsPost(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(fmt.Errorf("Failed creating project %q: %w", project.Name, err))
 	}
 
-	if d.rbac != nil {
-		err = d.rbac.AddProject(id, project.Name)
-		if err != nil {
-			return response.SmartError(err)
-		}
+	err = d.authorizer.AddProject(id, project.Name)
```

Comment:
Why does this use d.authorizer but the projectsGet function above use d.State().Authorizer?

Reply:
I've changed it to use `s.Authorizer` whenever possible.